### PR TITLE
Grid component

### PR DESCRIPTION
**Pull request by monoamro**

### CODE REVIEW

File: src/navigation/MainRouter.js [hunk 1]
```diff
@@ -18,13 +18,16 @@ const MainRouter = ({ userDetails }) => {
           description={about?.description}
         />
       </Route>
-      {galleries?.map((gallery, index) => {
+      <Route exact path={'/galleries/:galleryName'} >
+        <Gallery />
+      </Route>
+      {/* {galleries?.map((gallery, index) => {
```
Comment (widukin) [inline src/navigation/MainRouter.js:24]: Here is some commented  code. Could that be deleted?

File: src/shared/components/Wrapper/Styles.js [hunk 2]
```diff
@@ -2,8 +2,8 @@ import styled from 'styled-components';
 import { colour } from '../../styles/StyleConstants';
 
 export const StyledContainer = styled.div`
-  width: 100vw;
-  max-width: 1200px;
+  width: 99vw;
+  ${'' /* max-width: 1200px; */}
```
Comment (widukin) [inline src/shared/components/Wrapper/Styles.js:6]: what is happening here?

File: src/views/Gallery/index.js [hunk 3]
```diff
@@ -1,39 +1,35 @@
...
+
 
 const Gallery = ({ galleryId }) => {
   const [gallery, setGallery] = useState();
   const [images, setImages] = useState();
+  const { galleryName } = useParams();
 
   useEffect(() => {
-    Api.getGallery(galleryId)
+    Api.getGalleryByName(galleryName)
```
Comment (widukin) [inline src/views/Gallery/index.js:14]: Gallery names are not unique, what happens when two galleries have the same name?

**Review by widukin — CHANGES_REQUESTED** [review]
I left some comments on the code, please have a look

File: src/shared/components/Wrapper/Styles.js (hunk 2 above)
Comment (monoamro) [reply src/shared/components/Wrapper/Styles.js:6]: So that the site doesn't scroll right and left

File: src/views/Gallery/index.js (hunk 3 above)
Comment (monoamro) [reply src/views/Gallery/index.js:14]: Good catch, this should be figured out, we can make it in the scope of the subdomain name. Or just use the gallery id

**Pull request merged**